CONTRIBUTING.md

```diff
@@ -207,174 +208,185 @@ started with a basic development workflow. For other install modes see [Operator
+it to your Kubernetes cluster:
 
-   ```
-   $ BUNDLE_SRC_DIR=deploy/olm/bundle
-   $ operator-courier verify $BUNDLE_SRC_DIR
-   ```
-4. Retrieve a Quay.io token:
-   ```
-   $ QUAY_USERNAME=<your quay.io username>
-   $ QUAY_PASSWORD=<your quay.io password>
-   $ QUAY_URL=https://quay.io/cnr/api/v1/users/login
+```
+$ docker build -f starboard-catalog.Dockerfile -t starboard-catalog:dev .
+$ kind load docker-image starboard-catalog:dev
+```
 
-   $ QUAY_TOKEN=$(curl -s -H "Content-Type: application/json" -XPOST $QUAY_URL -d \
-     '{"user":{"username":"'"${QUAY_USERNAME}"'","password": "'"${QUAY_PASSWORD}"'"}}' |
-     jq -r .token)
-   ```
-5. Push the OLM bundle to Quay.io:
-   ```
-   $ QUAY_NAMESPACE=<quay.io namespace>
-   $ PACKAGE_NAME=starboard-operator
-   $ PACKAGE_VERSION=<next package version>
+### Register the Catalog Image
 
-   $ operator-courier push "$BUNDLE_SRC_DIR" "$QUAY_NAMESPACE" \
-     "$PACKAGE_NAME" "$PACKAGE_VERSION" "$QUAY_TOKEN"
-   ```
-6. Navigate to https://quay.io/application/$QUAY_USERNAME/starboard-operator?tab=settings and make the published
-   bundle public by clicking the **Make Public** button.
+Create a CatalogSource instance in the `olm` namespace to reference in the Operator catalog image that contains the
+Starboard Operator:
+
+```
+cat << EOF | kubectl apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: starboard-catalog
+  namespace: olm
+spec:
+  publisher: Starboard Maintainers
+  displayName: Starboard Catalog
+  sourceType: grpc
+  image: starboard-catalog:dev
+EOF
+```
 
+You can delete the default catalog that OLM ships with to avoid duplicate entries:
 
-### Install the OLM Bundle from Quay.io
+```
+$ kubectl delete catalogsource operatorhubio-catalog -n olm
+```
 
-1. Create the OperatorSource resource:
+Inspect the list of loaded packagemanifests on the system with the following command to filter for the Starboard Operator:
 
-   ```
-   QUAY_FULL_NAME=<your quay.io full name>
-   $ cat << EOF | kubectl apply -f -
-   apiVersion: operators.coreos.com/v1
-   kind: OperatorSource
-   metadata:
-     name: $QUAY_USERNAME-operators
-     namespace: marketplace
-   spec:
-     type: appregistry
-     endpoint: https://quay.io/cnr
-     displayName: "$QUAY_FULL_NAME Quay.io Applications"
-     publisher: "$QUAY_FULL_NAME"
-     registryNamespace: "$QUAY_USERNAME"
-   EOF
-   ```
+```console
+$ kubectl get packagemanifests
+NAME                 CATALOG             AGE
+starboard-operator   Starboard Catalog   97s
+```
 
-   An OperatorSource resource defines the external data store used to host operator bundles. In this case, you will be
-   defining an OperatorSource to point to your Quay.io account, which will provide access to its hosted OLM bundles.
+If the Starboard Operator appears in this list, the catalog was successfully parsed and it is now available to install.
 
-2. Create the OperatorGroup resource:
+### Create the OperatorGroup
```

Comment:
Is some of this duplicate of what's at https://aquasecurity.github.io/starboard/operator/installation/olm/ ? I guess here in the contribution guide it makes sense to talk about how to build and publish it but if from here on it's about running it, maybe refer to the documentation for that?

Reply:
Yup. It's duplicate. The only difference is the source catalog `starboard-catalog` instead of `operatorhubio-catalog` in the Subscription config (`spec.source`). I removed duplicate and added link to OLM installation guide.